tests/storage_test.go

```diff
@@ -253,5 +253,54 @@ var _ = Describe("Storage", func() {
 				close(done)
 			}, 400)
 		})
+
+		Context("With VM with two PVCs", func() {
+			BeforeEach(func() {
+				// Setup second PVC to use in this context
+				tests.CreatePvISCSI(tests.CustomISCSI, 1, false)
+				tests.CreatePVC(tests.CustomISCSI, false)
+			}, 120)
+
+			AfterEach(func() {
+				tests.DeletePVC(tests.CustomISCSI, false)
+				tests.DeletePV(tests.CustomISCSI, false)
+			}, 120)
+
+			It("should start vm multiple times", func() {
+				checkReadiness()
+
+				vm := tests.NewRandomVMWithPVC(tests.DiskAlpineISCSI)
+				tests.AddPVCDisk(vm, "disk1", "virtio", tests.DiskCustomISCSI)
+				vm.Spec.NodeSelector = map[string]string{"kubernetes.io/hostname": nodeName}
+
+				num := 3
+				By("Starting and stopping the VM number of times")
+				for i := 1; i <= num; i++ {
+					obj := RunVMAndExpectLaunch(vm, false, 120)
+
+					// Verify console on last iteration to verify the VM is still booting properly
+					// after being restarted multiple times
+					if i == num {
+						By("Checking that the VM console has expected output")
+						expecter, _, err := tests.NewConsoleExpecter(virtClient, vm, "serial0", 10*time.Second)
```

Comment:
Could you use the tests.LoggedInCirrosExpecter and use cirros for the vm?

Reply:
I could If I would be using cirros. This test is using Alpine linux. Therefore I cannot use logged in cirros.
I can write logged in Alpine, though :)

Reply:
You can, but you can also just switch to cirros, like mentioned above. (its the smallest and fastest booting). If you want to insist on using fdisk, it is also present in the cirros image.

Reply:
But you can also streamline the apline login in a testing.LoggedInAlpineExpecter..

Reply:
This would be really nice solution. I will do this in follow up patch. Doing it in this patch would introduce unnecessary changes.

The follow up patch will also change existing  alpine login expecters.

@rmohr Are you OK with it?